driver/flags_table.c

```diff
@@ -50,7 +50,11 @@ const struct ppm_name_value socket_families[] = {
 	{"AF_LOCAL", PPM_AF_LOCAL},
 	{"AF_UNIX", PPM_AF_UNIX},
 	{"AF_UNSPEC", PPM_AF_UNSPEC},
+#if defined(_WIN64) || defined(WIN64) || defined(_WIN32) || defined(WIN32)
+	{NULL, 0},
```

Comment:
We could just use the `{NULL, 0}` form unconditionally

Reply:
That's what I did initially. See the first commit in this PR
https://github.com/draios/sysdig/pull/1240/commits/d31a0659c3571d7561b5af7a98bfa971194b7e4c

But that caused a build failure because NULL is not recognized
https://api.travis-ci.com/v3/job/148757968/log.txt
~~~~
...
[ 46%] [32mBuilding C object userspace/libscap/CMakeFiles/scap.dir/__/__/driver/flags_table.c.o[0m
cd /home/travis/build/draios/sysdig/build/userspace/libscap && /usr/bin/gcc-4.8 -DHAS_CAPTURE -DK8S_DISABLE_THREAD -DPLATFORM_NAME=\"Linux\" -I/home/travis/build/draios/sysdig/common -I/home/travis/build/draios/sysdig/build/zlib-prefix/src/zlib  -Wall -ggdb -D_DEBUG   -o CMakeFiles/scap.dir/__/__/driver/flags_table.c.o   -c /home/travis/build/draios/sysdig/driver/flags_table.c
/home/travis/build/draios/sysdig/driver/flags_table.c:53:3: error: â€˜NULLâ€™ undeclared here (not in a function)
  {NULL, 0},
   ^
/home/travis/build/draios/sysdig/driver/flags_table.c:72:2: error: initializer element is not constant
  {NULL, 0},
  ^
/home/travis/build/draios/sysdig/driver/flags_table.c:72:2: error: (near initialization for â€˜file_flags[15].nameâ€™)
/home/travis/build/draios/sysdig/driver/flags_table.c:81:2: error: initializer element is not constant
  {NULL, 0},
…
~~~~~~~~~~~~
 
I could, of course, include an appropriate library (stddef.h say) or redefine NULL in flags_table.c
Do you have a preference?

Reply:
If `#include <stddef.h>` doesn't break e.g. the driver build, then that would be my preference. Otherwise maybe `{0, 0}` is acceptable?